flask/app/static/styles/index.css

```diff
@@ -1,99 +1,110 @@
-h1{
-    background-color: #bbb;
-    padding: 10px;
-    border-radius: 15px;
-    margin: 25px 15px;
+h1 {
+  background-color: #f4cc70;
+  color: #794518;
+  padding: 10px;
+  border-radius: 15px;
+  margin: 25px 15px;
 }
 
-.flex-container{
-    display: flex;
-    justify-content: space-evenly;
-    margin: 50px 75px;
+.mt-auto {
```

Comment:
I would recommend not linking custom styles to a Bootstrap class because you might want to use `mt-auto` elsewhere without the styles specified here.